_partials/_migrate_dual_write_step2.md

```diff
@@ -7,8 +7,20 @@ you must think about how to handle the failure to write to either the source or
 target database, and what mechanism you want to or can build to recover from
 such a failure.
 
+Should your time-series data have foreign-key references into a plain table,
```

Comment:
I fail to see when something like this could happen. When the non-TS data is migrated then all the IDs that are going to be referenced by the backfilled data should be already present in TARGET. No?

Reply:
_If_ the customer is _only_ dual-writing timeseries data, yes. If they intend to dual-write _both_ the referenced data and the timeseries data then this can happen.